pkg/apis/skupper/v2alpha1/types.go

```diff
@@ -486,6 +486,8 @@ type ConnectorSpec struct {
 	Selector        string            `json:"selector,omitempty"`
 	Port            int               `json:"port"`
 	TlsCredentials  string            `json:"tlsCredentials,omitempty"`
+	NoClientAuth    bool              `json:"noClientAuth,omitempty"`
```

Comment:
Question for my own education: how have you been doing this?

Are you writing go + generating the crd yaml defintions, vice versa, or is it just a manual process?

Reply:
I use a manual process, with the go and yaml edited separately. It may be possible to do some generation one way or the other, but I have not used that.